packages/app/templates/ui-extensions/projects/pos_ui/src/index.liquid

```diff
@@ -21,7 +21,7 @@ extend('Retail::SmartGrid::Tile', (root, api) => {
     subtitle: 'A test app',
     enabled: true,
     onPress: () => {
-      api.navigation.navigateToFullScreenModal();
+      api.smartGrid.presentModal();
```

Comment:
Unrelated to this line, but I feel like line 6, for the Tile, we should do something a little bit more creative than Hello and Hello 😂 . While we're here can you change it to title="My app" subtitle="SmartGrid extension" ?

Reply:
And let's use the same values for the "vanilla" package below